fetch_models.js

```diff
@@ -0,0 +1,163 @@
+import fetch from 'node-fetch';
+import fs from 'fs';
+import path from 'path';
+import ProgressBar from 'progress';
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+// import { HttpsProxyAgent } from 'https-proxy-agent';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const models = [
```

Comment:
This is great, seeing the entire list of models up-front, which makes it easier to know where to download them if needed (rather than inspecting each sample's code).